docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -59,6 +59,85 @@ automatically handle the migration to a per-machine installation.
 </TabItem>
 </Tabs>
 
+### Managed updates
+
+Teleport Connect supports [Teleport Client Tool Managed Updates](../upgrading/client-tools-managed-updates.mdx).
+When enabled in your cluster, the app checks for available updates at login,
+downloads them automatically, and prompts you to restart. On Windows and Linux,
+you may be asked to provide administrator credentials to complete the installation.
+
+You can also check for updates manually via "Check for Updates…" in the
+additional actions menu.
+
+<Admonition
+  type="warning"
+  title="Note"
+>
+  Managed updates are not supported for Linux `tar.gz` builds.
+</Admonition>
+
+#### Multiple clusters
+
+If you are connected to multiple clusters, Teleport Connect selects the highest client
+version that is compatible across all clusters according to [Teleport component compatibility rules](../upgrading/overview.mdx#component-compatibility).
+If clusters require incompatible versions, or if you want a different cluster
+to manage updates, you can manually choose the cluster in the
+"Check for Updates…" dialog, available in the additional actions menu.
+
+<Admonition
+  type="note"
+  title="Note"
+>
+  Multi-cluster support differs between Teleport Connect and the tsh/tctl CLI tools.
+  The CLI tools can switch versions on the fly to match the one required by each cluster.
+  Teleport Connect, as an installed desktop application, can only run a single version at a time.
+</Admonition>
+
+#### Managed updates configuration
+
+Like the CLI tools, Teleport Connect respects the `TELEPORT_CDN_BASE_URL` and
+`TELEPORT_TOOLS_VERSION` environment variables.
+
+* `TELEPORT_CDN_BASE_URL` lets you use custom builds or mirror the CDN in a private
+network (for example `https://example.com`).
+* `TELEPORT_TOOLS_VERSION` controls client tool updates:
+* Set to `off` to completely disable managed updates.
+* Set to a specific version (e.g. `18.0.1`) to override the cluster-provided version
+(for example, to work around a known issue).
+
+To use an environment variable with Teleport Connect, open your terminal and launch
+the app from there, providing the variable.
+It will apply only for that session, so you can test settings or override
+cluster-provided updates without affecting your system-wide configuration.
+For a permanent setup, follow the instructions below:
+
+<Tabs>
+  <TabItem label="macOS">
+    To set the variable for your current login session, open the Terminal and type:
+    ```console
+    launchctl set env TELEPORT_TOOLS_VERSION X.Y.Z
+    ```
```

Comment:
```suggestion
    ```code
    $ launchctl set env TELEPORT_TOOLS_VERSION X.Y.Z
    ```
```
To use our custom code component for commands